Tasks: publish task fails due to PackageVersion

### Description

There was a failure in the publish: https://github.com/ZCrewSoftware/ZCrew.Extensions.Tasks/actions/runs/21321973682/job/61373127480

```
/usr/share/dotnet/sdk/10.0.102/NuGet.Build.Tasks.Pack.targets(111,5): error MSB4018: The "GetPackOutputItemsTask" task failed unexpectedly. [/home/runner/work/ZCrew.Extensions.Tasks/ZCrew.Extensions.Tasks/src/ZCrew.Extensions.Tasks/ZCrew.Extensions.Tasks.csproj]
/usr/share/dotnet/sdk/10.0.102/NuGet.Build.Tasks.Pack.targets(111,5): error MSB4018: System.ArgumentException: PackageVersion string specified 'v1.0.0-preview.1' is invalid. [/home/runner/work/ZCrew.Extensions.Tasks/ZCrew.Extensions.Tasks/src/ZCrew.Extensions.Tasks/ZCrew.Extensions.Tasks.csproj]
/usr/share/dotnet/sdk/10.0.102/NuGet.Build.Tasks.Pack.targets(111,5): error MSB4018:    at NuGet.Build.Tasks.Pack.GetPackOutputItemsTask.Execute() [/home/runner/work/ZCrew.Extensions.Tasks/ZCrew.Extensions.Tasks/src/ZCrew.Extensions.Tasks/ZCrew.Extensions.Tasks.csproj]
/usr/share/dotnet/sdk/10.0.102/NuGet.Build.Tasks.Pack.targets(111,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Execute() [/home/runner/work/ZCrew.Extensions.Tasks/ZCrew.Extensions.Tasks/src/ZCrew.Extensions.Tasks/ZCrew.Extensions.Tasks.csproj]
/usr/share/dotnet/sdk/10.0.102/NuGet.Build.Tasks.Pack.targets(111,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(TaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [/home/runner/work/ZCrew.Extensions.Tasks/ZCrew.Extensions.Tasks/src/ZCrew.Extensions.Tasks/ZCrew.Extensions.Tasks.csproj]
```

### Expected Behavior

`dotnet pack -p:PackageVersion="v1.0.0-preview.1"` will fail, so strip the leading `v` so `dotnet pack -p:PackageVersion="1.0.0-preview.1"` is ran

### Steps To Reproduce

Deploy a new release.

### Additional Context

None.

--- a/.github/workflows/publish.yaml
+++ b/.github/workflows/publish.yaml
@@ -33,7 +33,9 @@ jobs:
         run: dotnet test --configuration Release --no-build --verbosity normal
 
       - name: Pack
-        run: dotnet pack --configuration Release --no-build --output ${{ env.NuGetDirectory }} -p:PackageVersion=${{ github.event.release.tag_name }}
+        env:
+          VERSION: ${{ github.event.release.tag_name }}
+        run: dotnet pack --configuration Release --no-build --output ${{ env.NuGetDirectory }} -p:PackageVersion=${VERSION#v}
 
       - name: NuGet login
         uses: NuGet/login@v1

